src/main/java/io/vertx/core/http/HttpClientOptions.java

```diff
@@ -138,6 +138,11 @@
    */
   public static final boolean DEFAULT_HTTP2_CLEAR_TEXT_UPGRADE = true;
 
+  /**
+   * Default WebSocket Masked bit is true as depicted by RFC. PerformingUnMasking will be false
+   */
+  public static final boolean DEFAULT_FRAME_UNMASKING = false;
```

Comment:
last but not least rename `DEFAULT_FRAME_UNMASKING` to `DEFAULT_SEND_UNMASKED_FRAMES`